### PR TITLE
Added new character helicopter

### PR DESCRIPTION
**Pull request by veehshiftz**

### CODE REVIEW

File: javascript/flapping.js [hunk 1]
```diff
@@ -84,6 +84,7 @@ function characterChange(layer) {
     hide_layer('bat'); 
     hide_layer('alien');
 	hide_layer('flappydino');
+	hide_layer('helicopter');
```
Comment (niccokunzmann) [inline javascript/flapping.js]: Please make the indentation correct in both lines: remove the tab and replace it by 4 spaces. 
The other line is a mistake that slipped through.

Comment (jig08) [inline javascript/flapping.js]: Use the same indentation @veehshiftz ?

**Comment by niccokunzmann** [comment]
I really like the helicopter jumping. It looks better than the jumping of the camel.

**Comment by veehshiftz** [comment]
I squashed the commits and changed the indentation

**Comment by niccokunzmann** [comment]
Awesome! Thanks! Also, you can add your name to the contributors file in the master branch, if you like.

**Pull request merged**